### ISSUE TITLE
No longer opening new maps after version 0.2.2489.11441

### ISSUE THREAD

**Issue by Windsinger1985** [allsyst3msg0/StationeersMapTrimmer#3]
Hi,

**Issue:**
Opening map shows a blank screen with "undefined" values

**Game client version:** 
0.2.2489.11441

**To recreate:**
- Create a new map, i took mars as example, base map, no custom.
- place some basic items.
- Save the map & close the game
- Try to open it in map trimmer. it loads for a few seconds then return to a blank screen.

**Comment by marccardinal** [comment]
Thanks for the report!

It looks like this might be the cause:

```
0.2.2485.11420 - Reduced the number of chunks saved to reduce save size and improve save/load speed.
```

I will have a look at the details of that change and report back.

**Comment by marccardinal** [comment]
Hello!

I spoke with a Stationeers dev and it looks like the terrain save file (world.bin) indeed changed format after 11420.

I am currently working on detecting the two different formats and handle the new format correctly.

Thanks for pointing it out!

**Comment by Windsinger1985** [comment]
No problem! 

Hope you're able to find a solution! thank for your time/dedication into making this possible for us!

**Comment by marccardinal** [comment]
This should have been fixed by [1.0.3](https://github.com/allsyst3msg0/StationeersMapTrimmer/releases/tag/1.0.3), let me know if that works on your side :)

**Comment by Windsinger1985** [comment]
Looks like it's all fixed!
It opens/saves without problems! thank you :)
